src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const elements = [...document.querySelectorAll('.population')];
+
+const values = elements.map((element) => {
+  const number = parseInt(element.textContent.replace(/,/g, ''), 10);
+
+  if (isNaN(number)) {
+    return;
+  }
+
+  return number;
+});
+
+const total = values.reduce((sum, value) => sum + value, 0);
+const average = Math.round(total / values.length);
```

Comment:
Dividing by `values.length` without filtering out `undefined` values can lead to incorrect averages. Ensure `values` only contains valid numbers.